css/theme.css

```diff
@@ -4633,7 +4633,6 @@ body {
   background:      -o-linear-gradient(90deg, #5C258D 10%, #4389A2 90%); /* Opera 11.10+ */
   background:         linear-gradient(90deg, #5C258D 10%, #4389A2 90%); /* W3C */
 
-  /*background: #663fb5 url(../img/bg-header.jpg) no-repeat center bottom;*/
```

Comment:
I found this commented line and I believe it's not needed. I hope I'm not missing some usage related to a weird browser behavior